src/Services/Authentication/UserLoginService.php

```diff
@@ -21,14 +22,33 @@ public function __construct(
     /**
      * @throws AuthenticationException
      */
-    public function loginUser(string $email, string $password): void
+    public function checkUser(string $email, string $password): string
```

Comment:
I don't understand why "check user" action is signing up an user and returns... string?

Reply:
changed